### PR TITLE
ATV resprite

### PR DESCRIPTION
**Pull request by Decortex**
## About the PR

Resprited old sprites
## Why / Balance

Because old sprites are boring

https://github.com/space-wizards/space-station-14/assets/122426300/cf62081a-8cab-4b66-ba82-f1b367811da9

## Media

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

### CODE REVIEW

File: Resources/Textures/Objects/Vehicles/atv.rsi/Vehicle.png
Comment (lzk228) [inline Resources/Textures/Objects/Vehicles/atv.rsi/Vehicle.png:1]: vehicle

File: Resources/Textures/Objects/Vehicles/atv.rsi/meta.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "y": 32
   },
   "license": "CC-BY-SA-3.0",
-  "copyright": "Taken from tgstation at https://github.com/tgstation/tgstation/commit/40d89d11ea4a5cb81d61dc1018b46f4e7d32c62a",
+  "copyright": "Taken from tgstation at https://github.com/tgstation/tgstation/commit/40d89d11ea4a5cb81d61dc1018b46f4e7d32c62a, resprited by decortes (discord)",
   "states": [
     {
       "name": "vehicle",
```
Comment (Simyon264) [inline Resources/Textures/Objects/Vehicles/atv.rsi/meta.json:11]: You renamed the file to `Vehicle.png` from `vehicle.png`. Not sure, but I believe this doesn't work.

Comment (Decortex) [reply Resources/Textures/Objects/Vehicles/atv.rsi/meta.json]: Done

Comment (brainfood1183) [reply Resources/Textures/Objects/Vehicles/atv.rsi/meta.json]: github struggles with changes between vehicle.png and Vehicle.png Might need to push a change to a completely different word before pushing the lowercase version. Unless you managed to fix it.

**Comment by TheShuEd** [comment]
Thanks for the contribution!

**Pull request merged**